diff --git a/node_modules/adofai-lib/dist/utils/analysisUtils.js b/node_modules/adofai-lib/dist/utils/analysisUtils.js
index 1d24d1f..271e250 100644
--- a/node_modules/adofai-lib/dist/utils/analysisUtils.js
+++ b/node_modules/adofai-lib/dist/utils/analysisUtils.js
@@ -1,4 +1,4 @@
-import { DLC_EVENT_TYPES, VfxTier, DECO_TYPES, DECO_ACTION_TYPES, VFX_EVENT_TYPES } from "../constants";
+import { DLC_EVENT_TYPES, VfxTier, DECO_TYPES, DECO_ACTION_TYPES, VFX_EVENT_TYPES } from "../constants.js";
 export function containsDLC(level) {
     return level.getActions().some(action => DLC_EVENT_TYPES.includes(action.eventType || ''));
 }
